bluebutton-data-pipeline-rif-extract/src/test/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/RifFilesProcessorTest.java

```diff
@@ -401,4 +403,88 @@ public void process27InpatientClaimRecords() {
+		OutpatientClaimGroup claimGroup = (OutpatientClaimGroup) rifRecordEvent.getRecord();
+		Assert.assertEquals(RifFilesProcessor.RECORD_FORMAT_VERSION, claimGroup.version);
+		Assert.assertEquals(RecordAction.INSERT, claimGroup.recordAction);
+		Assert.assertEquals("18", claimGroup.beneficiaryId);
+		Assert.assertEquals("8850923132", claimGroup.claimId);
+		Assert.assertEquals(new Character('W'), claimGroup.nearLineRecordIdCode);
+		Assert.assertEquals("40", claimGroup.claimTypeCode);
+		Assert.assertEquals(LocalDate.of(2016, 01, 24), claimGroup.dateFrom);
+		Assert.assertEquals(LocalDate.of(2016, 01, 24), claimGroup.dateThrough);
+		Assert.assertEquals("180013", claimGroup.providerNumber);
+		Assert.assertFalse(claimGroup.claimNonPaymentReasonCode.isPresent());
+		Assert.assertEquals(new BigDecimal("693.92"), claimGroup.paymentAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.primaryPayerPaidAmount);
+		Assert.assertEquals("KY", claimGroup.providerStateCode);
+		Assert.assertEquals("1043293608", claimGroup.organizationNpi);
+		Assert.assertEquals("1265494744", claimGroup.attendingPhysicianNpi);
+		Assert.assertEquals("1265494744", claimGroup.operatingPhysicianNpi);
+		Assert.assertFalse(claimGroup.otherPhysicianNpi.isPresent());
+		Assert.assertEquals("01", claimGroup.patientDischargeStatusCode);
+		Assert.assertEquals(new BigDecimal("6348.85"), claimGroup.totalChargeAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.deductibleAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.bloodDeductibleLiabilityAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.professionalComponentCharge);
+		Assert.assertEquals(new IcdCode(IcdVersion.ICD_10, "R0789"), claimGroup.diagnosisPrincipal);
+		Assert.assertEquals(2, claimGroup.diagnosesAdditional.size());
+		Assert.assertEquals(new IcdCode(IcdVersion.ICD_10, "R0789"), claimGroup.diagnosesAdditional.get(0));
+		Assert.assertEquals(new IcdCode(IcdVersion.ICD_10, "I10"), claimGroup.diagnosesAdditional.get(1));
+		Assert.assertEquals(new IcdCode(IcdVersion.ICD_10, "R079"), claimGroup.diagnosesReasonForVisit.get(0));
+		Assert.assertFalse(claimGroup.diagnosisFirstClaimExternal.isPresent());
+		Assert.assertEquals(25, claimGroup.lines.size());
+		// Verify one of the claim lines.
+		OutpatientClaimLine claimLine = claimGroup.lines.get(5);
+		Assert.assertEquals(new Integer(6), claimLine.lineNumber);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.lines.get(0).bloodDeductibleAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.lines.get(0).cashDeductibleAmount);
+		Assert.assertEquals(new BigDecimal("66.24"), claimGroup.lines.get(0).paymentAmount);
+		Assert.assertEquals(new BigDecimal("0"), claimGroup.lines.get(0).nonCoveredChargeAmount);
+
+	}
+
+	/**
+	 * Ensures that {@link RifFilesProcessor} can correctly handle
+	 * {@link StaticRifResource#SAMPLE_B_OUTPATIENT}.
+	 */
+	@Test
+	public void process340OutpatientClaimRecords() {
```

Comment:
No big deal, but I'd avoid method/test names like this, as they tend to fall out of sync as sample data is updated/replaced overtime.